server/routes/treeRouter.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const treeRouter = express.Router();
+const helper = require('./utils');
+const db = require('../database');
+
+treeRouter.get(
+  '/potential_matches',
+  helper.handlerWrapper(async (req, res) => {
+    const id = req.query.capture_id;
+    //maximum distance in meters between possible matches and tree in query
+    const distance = req.query.distance || 6;
+    const query =
+      'SELECT id, image_url as "imageUrl", latest_capture_id as "latestCaptureId", lat, lon, species_id as "speciesId", morphology, age, status, created_at as "createdAt", updated_at as "updatedAt" FROM treetracker.tree t1 LEFT JOIN treetracker.capture t2 ON ST_DWithin(t1.estimated_geographic_location, t2.estimated_geographic_location, $2) WHERE t1.id=$1 AND t2.id<>$1';
```

Comment:
I see the returning object will have camel cased attribute names. What is the standard we should follow? I prefer to use camelCase only for variable declarations and all objects to use snake case. We also depend on the snake case to match the db table column names in projects for persistence to work smoothly. Hence my suggestion.

Reply:
@deepwinter what is your thought on this.

Reply:
@arunbakt I do think that snake_case will make interactions with the database easier. Maybe we should follow that standard for object keys. I will change the files to reflect that standard. Thanks for the feedback!

Reply:
@luacmartins, I agree with you. All object keys we could decide to follow snake case and camelcase for variable names. I approved this PR, please merge it when you get a chance.